Jellyfin.Plugin.TelegramNotifier/NotificationFormatter.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using Jellyfin.Plugin.TelegramNotifier.Configuration;
+
+namespace Jellyfin.Plugin.TelegramNotifier
+{
+    public class NotificationFormatter
+    {
+        private readonly Dictionary<NotificationFilter.NotificationType, Func<UserConfiguration, string, string>> _prefixFormatters;
+        private readonly Dictionary<NotificationFilter.NotificationType, Func<UserConfiguration, string>> _suffixFormatters;
+
+        public NotificationFormatter()
+        {
+            _prefixFormatters = new Dictionary<NotificationFilter.NotificationType, Func<UserConfiguration, string, string>>
+            {
+                [NotificationFilter.NotificationType.ItemAdded] = (user, subtype) => subtype switch
+                {
+                    "ItemAddedMovie" => user.MessageItemAddedPrefix ?? $"🎬",
+                    "ItemAddedSerie" => user.MessageItemAddedPrefix ?? $"📺 [Serie]",
+                    "ItemAddedSeason" => user.MessageItemAddedPrefix ?? $"📺",
+                    "ItemAddedEpisode" => user.MessageItemAddedPrefix ?? $"📺",
+                    _ => user.MessageItemAddedPrefix ?? string.Empty
+                },
+                // Add more types and their prefixes as needed
```

Comment:
By merging your work we will need to adapt every notifier and their starting unicode.

Reply:
Yes, it led to update each notifier event but I'm glad you appreciate it.